common/planners/src/plan_table_create.rs

```diff
@@ -36,7 +35,7 @@ pub struct CreateTablePlan {
 
     pub table_meta: TableMeta,
 
-    pub cluster_keys: Vec<Expression>,
+    pub cluster_keys: Vec<String>,
```

Comment:
@zhyass I modify this to make it easier to be compatible with new planner, how do you like this?

Reply:
> @zhyass I modify this to make it easier to be compatible with new planner, how do you like this?

I think it's fine